internal/venice-common/src/main/java/com/linkedin/venice/pushmonitor/PushStatusDecider.java

```diff
@@ -28,15 +27,15 @@
 
 
 /**
- * Decide the offline push status by checking all of replicas statuses under different offline push strategies.
+ * Decide the offline push status by checking all replicas statuses under different offline push strategies.
  */
 public abstract class PushStatusDecider {
   private final Logger logger = LogManager.getLogger(PushStatusDecider.class);
   private static final String REASON_NOT_IN_EV = "not yet in EXTERNALVIEW";
   private static final String REASON_NOT_ENOUGH_PARTITIONS_IN_EV = "not enough partitions in EXTERNALVIEW";
   private static final String REASON_UNDER_REPLICATED = "does not have enough replicas";
 
-  private static Map<OfflinePushStrategy, PushStatusDecider> decidersMap = new HashMap<>();
+  private static final Map<OfflinePushStrategy, PushStatusDecider> decidersMap = new HashMap<>();
```

Comment:
I don't think this map is needed at all... We can add a `PushStatusDecider` property to the `OfflinePushStrategy` enum, and then get it from there directly, rather than from a map.

Reply:
Thanks for pointing out! That's a good catch. Yeah actually I found that it should also not be hard to merge PushStatusDecider with its corresponding OfflinePushStatusStrategy as it has very tight mapping, and for the common publish static method in PushStatusDecider class it can go to util class for example PushStatusMonitorUtils.
For this PR I only removed the map as you suggested, I plan to make the change as a separate PR and then 3rd PR trying to consolidate push monitor classes and other changes.